lib/node_modules/@stdlib/math/stats/kde2d/lib/kde2d.js

```diff
@@ -0,0 +1,149 @@
+'use strict';
+
+// MODULES //
+
+var ndarray = require( '@stdlib/ndarray' );
+var linspace = require( '@stdlib/math/utils/linspace' );
+var pdf = require( '@stdlib/math/base/dists/normal/pdf' );
+var setReadOnly = require( '@stdlib/utils/define-read-only-property' );
+var isNumericArray = require( '@stdlib/assert/is-numeric-array' );
+var isObject = require( '@stdlib/assert/is-object' );
+var pickBandwidth = require( './pick-bandwidth.js' );
+var range = require( './range.js' );
+var validate = require( './validate.js' );
+
+
+// MAIN //
+
+/**
+* Computes 2-dimensional normal kernel density estimate.
+*
+* @param {NumericArray} x - array of X values
+* @param {NumericArray} y - array of Y values
+* @param {Options} [options] - function options
+* @param {NumericArray} [options.h] - array of length 2 containing the bandwidth values for X and Y respectively.
+* @param {number} [options.n] - The number of partitions on the X and Y axises
+* @param {NumericArray} [options.xLim] - Array of length 2 containing the lower and upper limits of X upon which to create the grid
+* @param {NumericArray} [options.yLim] - Array of length 2 containing the lower and upper limits of Y upon which to create the grid
+* @throws {TypeError} first argument must be an array
+* @throws {TypeError} second argument must be an array
+* @throws {Error} first and second arguments must be same length
+* @throws {TypeError} options argument must be an object
+* @throws {TypeError} must provide valid options
+* @returns {Object} Object containing the gx, gy and z estimates
+* @example
```

Comment:
@frank113 Is your local environment setup to perform pre-push checks? Lint errors such as this one should have caused pushing code to a remote to fail.

Reply:
Frank, I presume the code passed linting since you had some older lint rules. Can you re-run `make init` from the top-level directory and check whether everything passes afterwards?

Context: I recently added a lint rule to ensure that `example` tags are preceded by an empty line.

Reply:
`make init` should have automatically run when he pulled down the most recent `develop`.